src/main/java/rx/exceptions/Exceptions.java

```diff
@@ -27,11 +30,26 @@ private Exceptions() {
     }
 
     /**
-     * @warn javadoc missing
+     * Convenience method to throw a {@code RuntimeException} and {@code Error} directly
+     * or wrap any other exception type into a {@code RuntimeException}.
+     * @param t the exception to throw directly or wrapped
      * @return because {@code propagate} itself throws an exception or error, this is a sort of phantom return
      *         value; {@code propagate} does not actually return anything
      */
     public static RuntimeException propagate(Throwable t) {
+        return propagate(t, null);
```

Comment:
Why do we have return values on these when they end up just throwing? I just noticed that. We can't change it, but it's very odd.